Makefile

```diff
@@ -1,13 +1,48 @@
 OADP_TEST_NAMESPACE ?= openshift-adp
-REGION ?= us-east-1
-PROVIDER ?= aws
-CLUSTER_PROFILE ?= aws
+CLUSTER_TYPE ?= aws
+
+# CONFIGS FOR CLOUD
+# bsl / blob storage cred dir
+OADP_CRED_DIR ?= /var/run/oadp-credentials
+# vsl / volume/cluster cred dir
+CLUSTER_PROFILE_DIR ?= /Users/drajds/.aws
+
+# bsl cred file
+OADP_CRED_FILE ?= ${OADP_CRED_DIR}/new-aws-credentials
+# vsl cred file
+CI_CRED_FILE ?= ${CLUSTER_PROFILE_DIR}/.awscred
```

Comment:
After much confusion debugging ci.. this probably should've been called VSL_CRED_FILE